R/outbreak_setup.R

```diff
@@ -31,6 +31,7 @@
 #'   k = 1.95,
 #'   prop.asym = 0
 #' )
+#' outbreak_setup
```

Comment:
```suggestion
```

Reply:
the previous line should emit some output, right? this would just print the function body

Reply:
Good spot, have fixed in 36ba51c49e018ac168c8c2407ce069bbecf143ba.